Mathlib/Data/Fintype/Quotient.lean

```diff
@@ -1,90 +1,239 @@
 /-
 Copyright (c) 2018 Mario Carneiro. All rights reserved.
 Released under Apache 2.0 license as described in the file LICENSE.
-Authors: Mario Carneiro
+Authors: Mario Carneiro, Yuyang Zhao
 -/
+import Mathlib.Data.List.Pi
```

Comment:
I'm a bit wary of adding extra imports here since this file does get imported by a big chunk of Mathlib. But looking at the import graph, most of those dependencies look like good splitting candidates anyway, so I think on balance it does not cause a bigger tangle than we already have.